sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -687,3 +687,46 @@ ALTER TABLE `form_clinical_notes` ADD `note_related_to` TEXT COMMENT 'Reference
 #IfMissingColumn form_care_plan note_related_to
 ALTER TABLE `form_care_plan` ADD `note_related_to` TEXT COMMENT 'Reference to lists id for note relationships(json)';
 #EndIf
+
+#IfNotTable insurance_type_codes
+CREATE TABLE `insurance_type_codes` (
+  `id` int(11) NOT NULL default '0',
+  `type` varchar(60) NOT NULL,
+  `claim_type` text(2) NOT NULL,
```

Comment:
make text bare if that is called for here. if only need a smaller element, would use varchar instead.

Reply:
Also, is id something meaningful (if I know this, can provide some guidance on it)? Meaning is it used somewhere and needs to correctly map to the type or claim_type?

Reply:
`id` should be a 2 digit key that is used and needs to map to both

Reply:
Is `claim_type` the 2 digit key or is `id` the 2 digit key or are both used as 2 digit keys in different circumstances?

Reply:
`id` is the 2 digit key